modules/table/src/main/java/org/apache/ignite/internal/table/AbstractTableView.java

```diff
@@ -68,15 +69,15 @@ protected <T> T sync(CompletableFuture<T> fut) {
     /**
      * Converts an internal exception to a public one.
      *
-     * @param th Internal exception.
+     * @param t Internal exception.
      * @return Public exception.
      */
-    protected IgniteException convertException(Throwable th) {
-        if (th instanceof IgniteException) {
-            return (IgniteException) th;
+    protected IgniteException convertException(Throwable t) {
```

Comment:
why you didn't use org.apache.ignite.lang.IgniteExceptionMapperUtil#mapToPublicException ?

Reply:
Fixed.